src/main/java/com/stripe/model/Person.java

```diff
@@ -132,16 +144,62 @@ protected String getInstanceUrl() {
     List<String> pastDue;
   }
 
-  // TODO: Move this to a top level class instead of duplicating
-  // with the LegalEntity resource
   @Getter
   @Setter
   @EqualsAndHashCode(callSuper = false)
   public static class Verification extends StripeObject {
     String details;
     String detailsCode;
-    String document;
-    String documentBack;
+    VerificationDocument document;
+    // TODO: Figure out how to handle the change in shape and deprecate the next 2
+    //String document;
+    //String documentBack;
     String status;
   }
+
+  @Getter
+  @Setter
+  @EqualsAndHashCode(callSuper = false)
+  public static class VerificationDocument extends StripeObject {
+    @Getter(lombok.AccessLevel.NONE) @Setter(lombok.AccessLevel.NONE) ExpandableField<File> back;
+    String details;
+    String detailsCode;
+    @Getter(lombok.AccessLevel.NONE) @Setter(lombok.AccessLevel.NONE) ExpandableField<File> front;
```

Comment:
Try changing these (and for `back` too) to just `AccessLevel.NONE`, without the `lombok.` prefix.

Reply:
Can you clarify why that's needed since other expandable fields have both?

Reply:
That's not the case AFAICT. On the latest `master`:
```
❯ ag "lombok\.AccessLevel\.NONE" --stats
0 matches
```